learn/on-premise/configure/config-users-auth-providers.md

```diff
@@ -24,24 +24,50 @@ The number of users that can be added is limited by the terms in License and the
 
 Following are the steps to add Users to the network by the SuperAdmin from WaveMaker Launchpad:
 
-- Select User on-boarding section at the left side of Launchpad.
-- At users section select Create User for creating the user.
-- Create a user by providing basic details like name ,email and password and select the prefer role for a user.
-- Roles
-- Super Admin - Access to Launchpad, Enterprise Network, Studio.
-- Enterprise Admin - Access to Enterprise Network and Studio.
-- Studio User - Access to Studio. All Developers will come under this.
+- Select Users on-boarding section at the left side of Launchpad.
+- At users section select Add New User for creating the user.
+- Create a user by providing basic details like name ,email and password.
+- The User created here will consider as a Studio User who can Access to Studio. All Developers will come under this.
+
```

Comment:
User will be part of Default/First Team automatically.
Add/Remove from Teams as needed